developer_docs/README.md

```diff
@@ -86,6 +86,8 @@ With the server running, you should be able to open `test/test.html` in a browse
 
 A complete guide to unit testing is beyond the scope of the p5.js documentation, but the short version is that any major changes or new features implemented in the source code contained in the `src/` directory should also be accompanied by test files in the `test/` directory that can be executed by Mocha to verify consistent behavior in all future versions of the library. When writing unit tests, use the [Chai.js reference](http://www.chaijs.com/api/assert/) as a guide for phrasing your assertion messages so that any errors caught by your tests in the future will be consistent and consequently easier for other developers to understand.
 
+The testing is done with the [mocha](http://visionmedia.github.io/mocha/) test framework with [mocha-chrome](https://github.com/shellscape/mocha-chrome#readme). 
```

Comment:
this section has been removed from the readme and moved over here: https://github.com/processing/p5.js/tree/master/developer_docs#unit-tests

Reply:
oops, i missed that. i'll try again...

Reply:
i rebase onto master. that change is now in 65f2f0c...

Reply:
👍

Reply:
except i typo'd. please stand by....

Reply:
now in b89bd7ac2b1f2e8605e3c77b882dbcdf855def90